### DISCUSSION TITLE
How to add button on a page?

### DISCUSSION THREAD

**Discussion by heysujal** [open-sauced-craftwork/browser-extension#6] [Q&A] [answered]
I followed all the tutorials and now I am trying to add a button on the current page. But I am not able to. I tried numerous blogs but it didn't work. How do I proceed?

**Comment by Code-Victor** [comment, marked as answer]
You'll have to use a [content script](https://developer.chrome.com/docs/extensions/mv3/content_scripts/). Check [here](https://crxjs.dev/vite-plugin/getting-started/vanilla-js/add-content-script) for how to do it in vanilla JS with CRX

**Reply by heysujal** [reply]
Thanks for the reply, I got it working already.

**Reply by vijaypotnuru** [reply]
Bro can you please tell me step by step process to start this assignment

**Reply by bdougie** [reply]
> Bro can you please tell me step by step process to start this assignment

The is a link in the [README](https://github.com/open-sauced-craftwork/browser-extension#task-instructions) to DEV post to help you get started.
